tests/unit/guildRole.test.ts

```diff
@@ -0,0 +1,111 @@
+import { createNewRole } from "../../src/typeDefinitions/discordMessage.types";
+import JSONResponse from "../../src/utils/JsonResponse";
+import * as response from "../../src/constants/responses";
+import { createGuildRole, addGroupRole } from "../../src/utils/guildRole";
+
+describe("createGuildRole", () => {
+  test("should return INTERNAL_SERVER_ERROR when response is not ok", async () => {
+    const mockEnv = {
+      DISCORD_GUILD_ID: "1234",
+      DISCORD_TOKEN: "abcd",
+    };
+    const mockBody: createNewRole = {
+      rolename: "test role",
+      mentionable: true,
```

Comment:
can we create fixtures for there please.

Reply:
sure